cohorts/1/README.md

```diff
@@ -0,0 +1,25 @@
+![Bits of Good](assets/header.png)
+# Cohort Hypertext Assassins
+
+### Our Motto
+Sparking joy one flame at a time \\(^ヮ^)/
+
+### Cohort Members
+* [Emily Layton](https://github.com/GTBitsOfGood/bootcamp/tree/fall-2020/hypertextAssassins-Emily)
+* [Marlene Garcia](https://github.com/GTBitsOfGood/bootcamp/tree/fall-2020/hypertextAssassins-marlene)
+* [Kirtana Mogili](https://github.com/GTBitsOfGood/bootcamp/tree/fall-2020/hypertextAssassins-kirtana)
+* [Eashan Sinha](https://github.com/GTBitsOfGood/bootcamp/tree/fall-2020/hypertextAssassins-Eashan)
+* [Carsen Miller](https://github.com/GTBitsOfGood/bootcamp/tree/fall-2020/hypertextAssassins-carsen)
+* [Isha Palakurthy](https://github.com/GTBitsOfGood/bootcamp/tree/fall-2020/hypertextAssassins-isha)
+
+### Our Dear Mascot: Calcifer
+![Our Dear Mascot: Calcifer](https://media.giphy.com/media/qKmWi9mfycx6E/giphy.gif)
+
+
+### License
+
+This repository is licensed under the [GNU GPLv3 License](/LICENSE.txt).
+
+
+### Our Motto
```

Comment:
Not sure if this should be here, since the motto is also listed above. But I might be totally wrong lol